package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/newjersey/njwds#readme",
   "dependencies": {
-    "@uswds/uswds": "3.0.1"
+    "@uswds/uswds": "3.5.0"
```

Comment:
✅  Confirming this is the latest release tag https://github.com/uswds/uswds/releases